.changeset/search-rankly-my-dear.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-catalog': patch
+'@backstage/plugin-search': patch
+'@backstage/plugin-techdocs': patch
+---
+
+In order to simplify analytics on top of the search experience in Backstage, the provided `<*ResultListItem />` component now captures a `discover` analytics event instead of a `click` event. This event includes the result rank as its `value` and, like a click, the URL/path clicked to as its `to` attribute.
```

Comment:
So I guess this mean that click events doesn't get captured anymore? Because of that change I wonder if this should either be changed to an optional replacement OR we should mark this as breaking? As this could be used for analysis already by adopters?